package.json

```diff
@@ -46,7 +46,7 @@
     "react-paginating": "^1.4.0",
     "react-scripts": "4.0.3",
     "react-select": "^4.3.1",
-    "react-simplemde-editor": "^4.1.5",
+    "react-simplemde-editor": "^5.0.2",
```

Comment:
Because of the changes in version 5 that are marked as "breaking", we should either test our usages of this or consider staying at version 4.1.5.

https://www.npmjs.com/package/react-simplemde-editor

Reply:
@zarsky-broad it looks like you wrote the code for editing snapshots. Can you help me with testing that the editor is still working correctly in that location?

Reply:
As long as you can edit and save, and the saving works, that's really all there is to see.

Reply:
@zarsky-broad can you provide steps on how I get to the Data Snapshot UI where this editor is present? I searched around in the Data tab but couldn't find anything about snapshots.

Reply:
Go to any workspace with a snapshot imported, click the info button for the snapshot in the data tab in the left bar, and you can edit the description.

Reply:
@zarsky-broad shared a workspace with me that included a snapshot. I was able to use the editor to update the snapshot description and save my changes.